projects/plugins/jetpack/changelog/remove-admin-menu-links-manager

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Not a big deal but I'd consider this `minor` seeing as we're not really fixing a bug, but rather deprecating some existing functionality.